### DISCUSSION TITLE
Feature: Allow us to disable threading in discussions

### DISCUSSION THREAD

**Discussion by vkbo** [community/community#109031] [Discussions]
### Select Topic Area

Product Feedback

### Body

The threaded feature of discussions is near useless, because people will just randomly create new threads in the middle of a conversation. This is not the fault of the users, but poor design of this feature. The quote feature starts a new thread by default, even when called from an existing thread, and reply by email does the same.

Adding to the frustration is that I, as an owner, cannot even move replies into the threads because I have no sensible moderation tools.

Considering how broken the threaded feature is, and considering how long these usability issues have gone unresolved, what I want more than anything is a way to just disable threaded comments altogether. Linear threads like in regular issues is preferable to this mess.

**Comment by mesupreneur** [comment]
It seems like you're experiencing frustration with the threaded feature of discussions on a platform. Here's how you can address this issue:

Feedback to Platform Support: Consider providing detailed feedback to the platform's support or development team regarding your frustrations with the threaded feature. Clearly outline the specific issues you're facing, such as the difficulty in managing threads and the lack of moderation tools. They may be able to take your feedback into account for future updates or improvements to the platform. 

Stay Informed about Updates: Keep an eye out for any updates or announcements from the platform regarding improvements or changes to the threaded feature. While it may take time for these issues to be addressed, staying informed can help you adapt your strategies accordingly.

**Reply by vkbo** [reply]
Replies like this are not helpful. I've already stated that this has gone unaddressed for years. What makes you think I'm not staying updated on changes?

The discussion feature is still very much a beta feature, and there are numerous topics on here pointing out all these challenges. Some are clearly bugs. If they are not prioritising fixing any of this, a way to opt out is preferable so the discussions can at least be used as a way to get noise out of the issues section.

**Reply by d-kleine** [reply]
> The threaded feature of discussions is near useless, because people will just randomly create new threads in the middle of a conversation. This is not the fault of the users, but poor design of this feature. The quote feature starts a new thread by default, even when called from an existing thread, and reply by email does the same.

I absolutely agree on that. But **I don't agree removing the threading in discussions** as this is super helpful for keeping track of discussions to a specific subject. I consider the issue more lies the bad usability, as you state, not seeing a clear visual difference and bad labeling if you are replying to a thread or set a new answer.
![grafik](https://github.com/community/community/assets/53251018/b0d13512-e17f-44b8-bf77-39421f9f0fd4)

Just a showcase for the need of improved usability could the the "quote reply" feature. When quoting a reply in a thread, why does this feature insert the quoted text in the "new answer" field instead of the "reply in thread" field?

https://github.com/community/community/assets/53251018/901c49d3-83f8-432a-906d-b3ecac55b65c

[If the video cannot be displayed for whatever reason, please reload the tab with CTRL+F5 (Windows users)]

**Reply by vkbo** [reply]
Another frustrating issue is that if you reply by email, it also ends up in the wrong place.

It would definitely be better if they resolved the issues. But it doesn't seem like they are, so a minimal option would be to disable threading and avoid all the broken stuff.

Another solution is to give us as owners some moderation tools to move replies to the correct place.

**Reply by vkbo** [reply]
Speaking of which, it may have been better to post your response as a new answer instead of as a reply to the AI generated spam answer. 😃

**Reply by d-kleine** [reply]
> It would definitely be better if they resolved the issues. But it doesn't seem like they are, so a minimal option would be to disable threading and avoid all the broken stuff.

I don't think that's a good idea because this would 
1. create even more messy discussions (even though there is indeed room for improvement)
2. most likely involve extra work

> Speaking of which, it may have been better to post your response as a new answer instead of as a reply to the AI generated spam answer. 😃

Before writing my response, I was actually thinking about that too 😃 I chose this thread because just to prove my point that threading is (sometimes, but not always) a helpful feature in discussions. I don't wanna get rid of threading just because the usability is not perfect. 

I think the best way to deal with these usability issues is by adding a screenshot/screencast demonstrating the problem and a constructive feedback how this problem could be resolved in a better way.

**Reply by vkbo** [reply]
> I think the best way to deal with these usability issues is by adding a screenshot/screencast demonstrating the problem and a constructive feedback how this problem could be resolved in a better way.

Sure, but people have literally been doing that for years to no effect.
